templates/scripts.js

```diff
@@ -0,0 +1,18 @@
+document.getElementById('url-form').addEventListener('submit', async function(event) {
```

Comment:
This file is duplicated

Reply:
Thank you for pointing that out! I will remove the duplicate `scripts.js` file and ensure that the correct one is in the `templates/static` directory.

I'll make the necessary changes and update the PR.